common/data/data.json

```diff
@@ -1142,11 +1142,11 @@
     "mask": ":mask:",
     "cornucopia": "<:cornucopia:1041714283924422836>",
     "rudolfs_red_nose": "<:rudolfs_red_nose:1047335258598670356>",
-    "tree": "<a:traditional_christmas_tree:1047335222708011058>",
+    "christmas_tree": "<a:traditional_christmas_tree:1047335222708011058>",
     "mistletoe": "<:mistletoe:1047336566638526545>",
     "inflate_santa": "<:inflate_santa:1047337111151464458>",
     "tree_globe": "<:tree_globe:1047337839068729364>",
-     "s_sack": "<:s_sack:1047338463097266217>",
+     "santa_sack": "<:santa_sack:1047338463097266217>",
```

Comment:
Remove this weird space please
![image](https://user-images.githubusercontent.com/38477514/204808277-ad690565-ec82-465f-99d5-5dc5ac741688.png)